### DISCUSSION TITLE
toVariablePayload - type unclear

### DISCUSSION THREAD

**Discussion by DavidMarom** [grafana/grafana#41661] [Plugins]
```dispatch(setCurrentVariableValue(
  toVariablePayload(
    {
      type: 'custom',
      id: 'aaa'
    },
    ???  <---  I'M STRUGGLING WITH THE TYPE HERE - HELP PLEASE
  )
));

**Comment by DavidMarom** [comment]
ok, I have found a better way to do it using getLocationSrv().update

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
